### ISSUE TITLE
TableView follow up: Merged cells

### ISSUE THREAD

**Issue by Piinks** [flutter/flutter#131224]
[Design Doc](https://docs.google.com/document/d/1UekXjG_VKmWYbsxDEzMqTb7F-6oUr05v998n5IqtVWs/edit?usp=sharing)

~Blocked on https://github.com/flutter/flutter/issues/133089~

Merged cells were omitted from the original PR for easier review. 
Once https://github.com/flutter/packages/pull/4536 lands, we can follow up with another PR to add support for merged cells.

Also:
- https://github.com/flutter/flutter/issues/21594

**Comment by IchordeDionysos** [comment]
Really looking forward to this! 😍 
What's the current status of it?

**Comment by Piinks** [comment]
This is currently blocked on https://github.com/flutter/flutter/issues/133089. I hav a PR out to fix that issue (https://github.com/flutter/flutter/pull/133581), but after it lands it will need to roll to stable before we can use it in the TableView.

I am looking into other options right now, but I think the current plan to implement merged cells with a ParentDataWidget is the best approach.

**Comment by uua-kirkcharlesniv** [comment]
Hello! First of all, great package and thank you for your contributions.

Is this feature **(merged cells)** now supported in `two_dimensional_scrollables` package? I just saw that https://github.com/flutter/flutter/pull/133581 and https://github.com/flutter/flutter/pull/133581 are already closed and merged.

Any updates would be nice, thanks!

**Comment by fanchou** [comment]
We are one last step away from using this great package.
